src/Microsoft.Health.Dicom.Functions/Microsoft.Health.Dicom.Functions.csproj

```diff
@@ -2,20 +2,19 @@
 
   <PropertyGroup>
     <AzureFunctionsVersion>v3</AzureFunctionsVersion>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFrameworks>netcoreapp3.1</TargetFrameworks>
   </PropertyGroup>
 
+  <ItemGroup>
+    <FrameworkReference Include="Microsoft.AspNetCore.App" />
```

Comment:
why replace those assemblies with Framework Microsoft.AspNetCore.App?

Reply:
Ultimately we're going to use ASP.NET Core components for our HTTP triggers. As such, we're essentially an ASP.NET Core project being hosted by someone else's ASP.NET Core project. So I think this framework reference makes it clearer this is an ASP.NET Core project and will automatically import all of the necessary dependencies.